src/Views/scstormrunoff.html

```diff
@@ -32,13 +33,63 @@ <h4 class="modal-title hidden-print">Study Area Storm Event Runoff Summary</h4>
                                 <label ng-if="vm.validateForm(BohmanRuralForm)">&nbsp;&nbsp;</label> 
-                            </form>  -->
-                            <div> Coming Soon. </div>
-                            <br>
+                            </form>
+                            <div ng-if="vm.showRuralResults">
+                                <div style="position:relative;">
+                                    <p class="fullwidth alignc hidden-print" style="font-size: 16px;"><b>USGS SC Flood Hydrograph for Rural Watersheds using {{vm.SelectedAEP.name}} AEP</b></p>
+                                    <div class="wim-alert-warning">{{vm.warningMessages}}</div>
+                                    <div class="well">
+                                        <h3>Parameters</h3>
+                                        <table id="BohmanRuralParameterTable" class="table table-condensed smalltext">
+                                            <tbody>
+                                                <tr>
+                                                    <td>Clicked Point (Latitude, Longitude):</td>
+                                                    <td><b>{{vm.studyAreaService.selectedStudyArea.Pourpoint.Latitude.toUSGSvalue()}}, {{vm.studyAreaService.selectedStudyArea.Pourpoint.Longitude.toUSGSvalue()}}</b></td>
+                                                </tr>
+                                                <tr>
+                                                    <td>Drainage Area (square miles):</td>
+                                                    <td><b>{{vm.drainageArea}}</b></td>
+                                                </tr>
+                                            </tbody>
+                                        </table>
+                                        <h3>Summary</h3>
+                                        <table id="BohmanRuralSummaryTable" class="table table-condensed smalltext">
+                                            <tbody>
+                                                <tr>
+                                                    <td>Weighted Runoff Volume (inches):</td>
+                                                    <td><b>{{vm.ReportData.BohmanRural1989.WeightedRunoff.toUSGSvalue()}}</b></td>
+                                                </tr>
+                                            </tbody>
+                                        </table>
+        
+                                        <div style='width:650px; margin:auto;' class="page_break" id="legend">
+                                            <nvd3 options='vm.ReportOptions' data='vm.ReportData.BohmanRural1989.Graph'></nvd3>
+                                        </div>
+        
+                                        <h3>Tabular Hydrograph</h3>
```

Comment:
Do you think the Tabular Hydrograph on the Bohman tabs should be collapsed? Or perhaps given the option to collapse it, but it's open by default? I could go either way. We collapse the table in the Synthetic tab because it's so large.

Reply:
Ill make them collapsed - I like the idea they all look and work in the same way.